ui/src/app/model/Conv.ts

```diff
@@ -0,0 +1,134 @@
+export default interface IConv {
+  mode: number
+  SpSchema: Record<string, ICreateTable>
```

Comment:
Based on the style guide : https://ts.dev/style/#identifiers , rename all the properties in interface to follow lower camel case

Reply:
Style guide https://ts.dev/style/#naming-style also mentions that its not recommended to mark interfaces specially (IMyInterface or MyFooInterface) unless it's idiomatic in its environment. Please have a look.

Reply:
Added to backlog as it may require changes in the backend also. 
Backlog no - 51